templates/treat_ad.yaml

```diff
@@ -0,0 +1,61 @@
+Structural_Biology_Core_Folder: &struct_bio_folder
+  name: Structural_Biology_Core
+  type: Folder
+  next_level:
+    - Antibody_Validation_Data
+    - Assay_Protocols
+    - Assay_Results
+    - Cell_Lines
+    - Chemotypes
+    - Crystal_Structures
+    - Expression_Constructs
+    - Expression_Data
+    - Fragment_Screening_Results
+    - Lab_Notebooks
+    - Protein
+    - Protein_Expression_Methods
+    - TEP
+    
+Assay_Core_Folder: &assay_folder
+  name: Assay_Core
+  type: Folder
+  next_level:
+    - Assay_Results
+    - Compounds
+    - HTS_Protocols
+    - HTS_Results
+    - Isogenic_Cell_Line
+    - Lab_Notebooks
+    - Off_Target_Potency
+    - Secondary_Screen_Protocols
+    - Secondary_Screen_Result
+    
+MedChem_Core_Folder: &medchem_folder
+  name: MedChem_Core
+  type: Folder
+  next_level:
+    - Chemical_Probe_Data
+    - Chemical_Provenance
+    - Lab_Notebooks
+    - Structure_Activity_Relationship
+
+Gene_Folder: &gene_folder
```

Comment:
Is this missing?
```
name:
type:
```
or is the folder name `Gene_Folder` and default type is `Folder`?

Reply:
Type defaults to "Folder" while the name is described by the list that has the names "MSN" and "CD44" for example! ☺️